scripts/generate-opds.php

```diff
@@ -1,19 +1,24 @@
 <?
-$contentFiles = explode("\n", trim(shell_exec('find /standardebooks.org/www/ebooks/ -name "content.opf" | sort') ?? ''));
+$longopts = array("webroot:", "weburl:");
+$options = getopt("", $longopts);
+$webRoot = $options["webroot"] ?? "/standardebooks.org";
+$webUrl = $options["weburl"] ?? "https://standardebooks.org";
+
+$contentFiles = explode("\n", trim(shell_exec('find ' . $webRoot . '/www/ebooks/ -name "content.opf" | sort') ?? ''));
```

Comment:
Make sure to use escapeshellarg() on any user input passed to shell_exec(). In this case we would do `escapeshellarg($webRoot . '/www/ebooks/')`